generator/packets.py

```diff
@@ -50,14 +51,38 @@
         ["wheelSpeed2",       32, [-50000., 50000.], "wheelSpeed2"],
         ["wheelSpeed3",       32, [-50000., 50000.], "wheelSpeed3"],
         ["wheelSpeed4",       32, [-50000., 50000.], "wheelSpeed4"],
+        ["bodyXIntegral",     8, None, "Integral value from the PID for body_x"],
```

Comment:
Should these not be float values? With this, they can only take up integer values between 0 and 255